build/vsts_build.yaml

```diff
@@ -212,6 +212,14 @@ phases:
       msbuildArguments: "/p:BuildNumber=$(BuildNumber) /p:IsVsixBuild=true"
     condition: " and(succeeded(), eq(variables['BuildRTM'], 'false'))"
 
+  - task: PowerShell@1
```

Comment:
Why?

This is NuGet's internal CI configuration. 
We removed the e2e test data package generation on purpose. 

The idea is that if needed, the test data package will be updated manually. 

See https://github.com/NuGet/NuGet.Client/pull/2804.

Reply:
thanks